### ISSUE TITLE
Add Print Button to my project Page

### ISSUE THREAD

**Issue by Biuwa** [hackforla/tdm-calculator#1345]
### Overview
We need to add a print button function to the `my project` page. The purpose of this is so users do not have to navigate to page 5 to print the summary of their project

### Action Items

- [x] create mockups 
- [x] Review with the team
- [x] Review with stakeholders

**Comment by IMHieuVo** [comment]
@Biuwa 

Here is the mockup for the Project page.
Figma file: https://www.figma.com/file/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=1054%3A846&t=fYW63sgj4E9ItZsx-1

I want to note a few things:
- I updated the typefront into the new design system.
- I noticed that the original table had smaller font so it was able to fit more headers.
- I put the print/download icon between the copy and trash but there isn't a strong inclination on the ordering.

Mockup:
![HV - Saved Projects](https://user-images.githubusercontent.com/117132709/231084298-1b3a7a3d-269d-445f-89e8-9e256f938f14.png)

Existing, for reference:
![Existing Project Page, screenshot](https://user-images.githubusercontent.com/117132709/231084288-563a04cc-7a7e-40a8-9f31-0954be7d185a.png)
